docs/source/server_configuration.rst

```diff
@@ -37,7 +37,7 @@ To create user accounts, one can use the ``add-users`` command of the
 
 .. code:: bash
 
-    asreview auth-tool add-users
+    asreview auth-tool add-users --db-uri=sqlite:////path/example.sqlite
```

Comment:
This is not required isn't it?

Reply:
No, but I think it makes sense to explicitly add examples. I don't like setting env variable before you run a CLI script.  All sub commands, list-projects being a exception, require access to the auth database. Then provide it as a parameter. I am afraid people will miss the ENV option because people usually don't read.

Reply:
But the default option is independent of setting the ENV var. I do understand the idea however. Especially when taking your advanced Docker setup into account. It starts to become useful to move the advanced Docker to a standalone repo and write extensive documentation. In my recent PR on the docs, I focussed on the simple Docker setup (or not even using Docker). There, I assume most people don't change the location of the database.